docs/usage/self-hosted-configuration.md

```diff
@@ -92,6 +92,10 @@ If you configure this to be different to the `baseDir`, it means you can have on
 
 Set to `false` to prevent usage of `--ignore-platform-reqs` in the Composer package manager.
 
+## customEnvVariables
+
+Custom environment variables for child processes and sidecar Docker containers. This configuration will be applied after all other environment variables so that it can be used to override defaults.
```

Comment:
```suggestion
This configuration will be applied after all other environment variables so that it can be used to override defaults.
```

Otherwise it will be duplicated once combined with the definitions